build/repo.targets

```diff
@@ -115,21 +115,21 @@
   </Target>
 
   <ItemGroup>
-    <JavaBuildFiles Include="signalr-client-$(JavaClientVersion)-preview1.jar;signalr-client-$(JavaClientVersion)-preview1-javadoc.jar;signalr-client-$(JavaClientVersion)-preview1-sources.jar;signalr-client-$(JavaClientVersion)-preview1.pom"/>
+    <JavaBuildFiles Include="signalr-client-$(JavaClientVersion)-$(VersionSuffix).jar;signalr-client-$(JavaClientVersion)-$(VersionSuffix)-javadoc.jar;signalr-client-$(JavaClientVersion)-$(VersionSuffix)-sources.jar;signalr-client-$(JavaClientVersion)-$(VersionSuffix).pom"/>
```

Comment:
FYI - this is going to result in `signalr-client-0.1.0-preview1-t000.jar` because VersionSuffix is appended with `$(BuildNumber)` in version.props.

An alternative to this is to make `$(JavaClientVersion)` contain the full version string, so all you need here is `signalr-client-$(JavaClientVersion).jar`


https://github.com/aspnet/SignalR/blob/e03166252578dca884f9b9d5c1d8a632de97754a/version.props#L11